.github/workflows/ci.yml

```
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Image version'
+        required: true
+env:
+  LOGIN_SERVICE_IMAGE_NAME: "kioku_login"
+  REGISTER_SERVICE_IMAGE_NAME: "kioku_register"
+  FRONTEND_SERVICE_IMAGE_NAME: "kioku_frontend"
+
+jobs:
+  build_push_and_deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set outputs
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+      - name: Check outputs
+        run: echo "Commit - ${{ steps.vars.outputs.sha_short }}"
+
+      - name: Login to GitHub Packages
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build Frontend
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./frontend/Dockerfile
```

Did you test this?
I think it should be:
```suggestion
          context: ./frontend
          file: Dockerfile
```
Otherwise the instruction of the Dockerfile would be (metaphorically)
```dockerfile
COPY go.mod go.sum ./
```
instead of
```dockerfile
COPY ./frontend/go.mod ./frontend/go.sum ./
```

---

I think you are right.
But no, I have not tested this before